### PR TITLE
Check i2c

### PR DESCRIPTION
**Pull request by jonl40**

### CODE REVIEW

File: test-i2c/test-i2c.ino [hunk 1]
```diff
@@ -1,4 +1,4 @@
-//imports
+ //imports
```
Comment (buzuloiu) [inline test-i2c/test-i2c.ino]: this space shouldn't be here

**Comment by jonl40** [comment]
imu locks up. Also, i'm not to sure where to put set axis with tare, and software reset.

File: test-i2c/Yost.h [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+     byte tempArray[4]; // {Return Len: 4 bytes} {Return Data Details: float}
+     float tempFloat;
+   } temp;
+
+   union euler_tag {
+     byte eulerArray[4]; // {Return Len 12 bytes} {Return Data Details: Euler Angles(float x3)}
+     float eulerFloat;
+    } euler;
+
+   //float angle_euler[3]; // store each float (every 4 bytes) into an index of the xyz_euler array
```
Comment (buzuloiu) [inline test-i2c/Yost.h:74]: Should this be commented out?

File: test-i2c/IMU_v2.ino [hunk 3]
```diff
@@ -0,0 +1,141 @@
+//Component: Yost 3-Space™ LX Embedded part number TSS-LX
+
+#include "Yost.h"
+#include <Wire.h>
+
+#define TRUE 1
+#define FALSE 0
```
Comment (buzuloiu) [inline test-i2c/IMU_v2.ino:7]: I don't think you need to define these in Arduino, I think `true` and `false` are already mapped to `1` and `0`

File: IMU_v2/IMU_v2.ino [hunk 4]
```diff
@@ -0,0 +1,134 @@
+//Component: Yost 3-Space™ LX Embedded part number TSS-LX
```
Comment (buzuloiu) [inline IMU_v2/IMU_v2.ino:1]: - I think you can delete this whole folder if the `test-i2c` folder is the one you want to use

File: test-i2c/test-i2c.ino [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
```
Comment (buzuloiu) [inline test-i2c/test-i2c.ino:1]: - you should copy the contents of `IMU_V2.ino` into this file and delete `IMU_V2.ino`,  `.ino` files should be in a folder with the same name as them

**Review by buzuloiu — COMMENTED** [review]
This looks good, just needs some cleaning up

File: test-i2c/Yost.h (hunk 2 above)
Comment (jonl40) [reply test-i2c/Yost.h:74]: It should be commented out. I'll remove it and make a new pull request

**Pull request merged**